docs/stellar-disbursement-platform/getting-started.mdx

```diff
@@ -235,26 +235,36 @@ In `docker-compose-sdp-anchor.yml`, update the following:
 <CodeExample>
 
 ```yaml
-# the public key of the account used for authentication
-# it should be the same public key for both attributes
-SEP10_ACCOUNT:
+# the public key of the Stellar account used for SEP-10 authentication
 SEP10_SIGNING_PUBLIC_KEY:
-# the private key of the account used for authentication
-# it should be the same secret key for both attributes
+#
+# the private key of the Stellar account used for SEP-10 authentication
+# it should be the same secret key for both attributes, in SDP and AP
```

Comment:
instead of SDP and AP, probably use full names? Stellar Disbursement Platform and Anchor Platform

Reply:
Addressed in https://github.com/stellar/stellar-docs/pull/239/commits/26f2f7ff0bcc7ca3d1f8d59927095e26e9e4d379.